rgw/v2/tests/s3_swift/multisite_configs/test_lc_cloud_transition_ibm_retain_false.yaml

```diff
@@ -0,0 +1,43 @@
+#test_bucket_lifecycle_object_expiration_transition.py
+#CEPH-83581973,CEPH-83581975
+config:
+  user_count: 1
+  bucket_count: 1
+  objects_count: 10
+  parallel_lc: False
+  test_lc_transition: True
+  enable_resharding: True
+  sharding_type: manual
+  shards: 211
+  pool_name: data.cold
+  storage_class: cold
+  ec_pool_transition: False
+  multiple_transitions: True
+  two_pool_transition: False
+  second_pool_name: data.glacier
+  second_storage_class: glacier
+  objects_size_range:
+    min: 5
+    max: 15
+  test_ops:
+    create_bucket: true
+    create_object: true
+    enable_versioning: true
+    version_count: 3
+    delete_marker: false
+    test_cloud_transition: true
+    test_pool_transition: false
+    test_ibm_cloud_transition: true
+    test_aws_cloud_transition: false
+    test_retain_head: false
```

Comment:
@viduship
could you make use of actual_lc_days config instead of sleeping for 20 days by default if the lc takes effect after 2 days
https://github.com/red-hat-storage/ceph-qe-scripts/blob/master/rgw/v2/tests/s3_swift/configs/test_lc_rule_conflict_transition_actions.yaml#L26